contracts/adapters/FixedRebalanceExtension.sol

```diff
@@ -47,34 +47,25 @@ contract FixedRebalanceExtension is BaseExtension {
     using SafeMath for uint256;
 
 
-    struct CallArguments {
-        address setToken;
-        uint16 currencyId;
-        uint40 maturity;
-        uint256 minMintAmount;
-        address sendToken;
-        uint256 sendAmount;
-    }
-
     // /* ============ Events ============ */
 
     event AllocationsUpdated(uint256[] _maturities, uint256[] _allocations);
     event MaturityStatusSet(uint256 _maturity, bool _newStatus);
 
     // /* ============ State Variables ============ */
 
-    mapping(uint256 => bool) internal isValidMaturity;       // Mapping of valid maturities
-    uint256[] validMaturities;                               // Array of valid maturities
-    ISetToken setToken;                                      // Instance of leverage token
-    INotionalTradeModule notionalTradeModule;                // Instance of leverage token
-    INotionalProxy notionalProxy;                            
-    IWrappedfCashFactory wrappedfCashFactory;                // Instance of leverage token
-    address underlyingToken;
-    address assetToken;
-    uint256[] maturities;                                    // Array of relative maturities in seconds (i.e. 3 months / 6 months)
-    uint256[] allocations;                                   // Relative allocations 
-    uint16 public currencyId;
-    bool tradeViaUnderlying;
+    uint256[] internal maturities;                               // Array of relative maturities in seconds (i.e. 3 months / 6 months)
+    uint256[] internal allocations;                              // Relative allocations 
+
+    ISetToken internal immutable setToken;                       // Instance of leverage token
```

Comment:
I'd consider making this public so we don't have to look at constructor args to know what Product this is for.

Reply:
Good point. Done